core/trino-main/src/test/java/io/trino/sql/TestLikeFunctions.java

```diff
@@ -254,4 +261,50 @@ public void testUnescapeValidLikePattern()
         assertEquals(unescapeLiteralLikePattern(utf8Slice("a##bc#_"), Optional.of(utf8Slice("#"))), utf8Slice("a#bc_"));
         assertEquals(unescapeLiteralLikePattern(utf8Slice("a###_bc"), Optional.of(utf8Slice("#"))), utf8Slice("a#_bc"));
     }
+
+    @Test
+    public void testRegexInterruptible()
+            throws Exception
+    {
+        String pattern = "\\((.*,)+(.*\\))";
+        String source = Files.readString(Paths.get(EXTRA_LONG_SOURCE_PATH));
```

Comment:
`Files.readString` approach works only when the resource is a real file. If it is packaged as a JAR this approach is not going to work. Please use `Resources.toString(Resources.getResource(...), UTF_8)` (here and in other place)

Reply:
Needs to be addressed